presto-docs/src/main/sphinx/installation/verifier.rst

```diff
@@ -2,9 +2,8 @@
 Presto Verifier
 ===============
 
-The Presto Verifier can be used to test Presto against another database (such as MySQL),
-or to test two Presto clusters against each other. We use it to continuously test trunk
-against the previous release while developing Presto.
+The Presto Verifier can be used to test two versions Presto against each other.
+We use it to continuously test trunk against the previous release while developing Presto.
```

Comment:
Who's we here? The documentation shouldn't reference facebook practice?

Reply:
Removed